### PR TITLE
Emulates blossom deployment with docker-compose

### PR DESCRIPTION
**Pull request by TheLonelyGhost**
## Description:

For closer emulation to what is running on prod, here's a `docker-compose` manifest and associated files for running blossom entirely locally using Docker and `docker-compose` as an orchestrator.

## Checklist:

- [x] Code Quality
- [x] Pep-8
- [x] Tests (if applicable)
- [x] Success Criteria Met
- [x] Inline Documentation
- [x] Wiki Documentation (if applicable)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+version: '3'
+services:
+  db:
+    image: postgres:11.5-alpine
+    environment:
+      POSTGRES_DB: 'mydb'
+      POSTGRES_PASSWORD: 'hunter2'
+      POSTGRES_USER: 'roto_rooter'
```
Comment (MaxVanDeursen) [inline docker-compose.yml:9]: Shouldn't we go for a bit more Blossom-like credentials?

**Review by MaxVanDeursen — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (TheLonelyGhost) [reply docker-compose.yml:9]: Probably, but that was to test that we're actually pulling in values from environment variables and showing you could make the creds _anything_. And that's important from a security standpoint if we need to quickly pivot.

Here's an example: If username/password get owned, rotating just the password is going to leave it open to brute force attack. They know the first factor (username) and they can repeatedly guess at the second (password). Eventually they'll guess right. Will we notice before that happens and lock down access? Hopefully. I'd rather not make that wager in the first place.

**Review by itsthejoker — APPROVED** [review]
LGTM

File: docker-compose.yml (hunk 1 above)
Comment (MaxVanDeursen) [reply docker-compose.yml:9]: That makes sense, my remark was indeed about the default values in code :)

**Review by MaxVanDeursen — APPROVED** [review]

**Comment by MaxVanDeursen** [comment]
image

**Pull request merged**